src/hooks/useAuth.tsx

```diff
@@ -7,7 +7,7 @@ import {
 } from "react";
 import { useLocation } from "wouter";
 import { AuthRequest, AuthResponse } from "@/types/auth";
-import { useMutation, useQueryClient } from "react-query";
+import { useMutation, useQueryClient } from "@tanstack/react-query";
```

Comment:
[codemod](https://tanstack.com/query/v4/docs/guides/migrating-to-react-query-4#codemod) existed, but an error occurred when converting it, so I converted  it manually.

```sh
 ERR src/hooks/useAuth.tsx Transformation error (Missing semicolon. (14:4))
SyntaxError: Missing semicolon. (14:4)
    at instantiate (/Users/jinma/.npm/_npx/c5e6d14ea42cfa80/node_modules/@babel/parser/src/parse-error/credentials.ts:62:21)
    at toParseError (/Users/jinma/.npm/_npx/c5e6d14ea42cfa80/node_modules/@babel/parser/src/parse-error.ts:60:12)
    at JSXParserMixin.raise (/Users/jinma/.npm/_npx/c5e6d14ea42cfa80/node_modules/@babel/parser/src/tokenizer/index.ts:1474:19)
    at JSXParserMixin.semicolon (/Users/jinma/.npm/_npx/c5e6d14ea42cfa80/node_modules/@babel/parser/src/parser/util.ts:142:10)
    at JSXParserMixin.parseExpressionStatement (/Users/jinma/.npm/_npx/c5e6d14ea42cfa80/node_modules/@babel/parser/src/parser/statement.ts:1162:10)
    at JSXParserMixin.parseStatementContent (/Users/jinma/.npm/_npx/c5e6d14ea42cfa80/node_modules/@babel/parser/src/parser/statement.ts:555:19)
    at JSXParserMixin.parseStatement (/Users/jinma/.npm/_npx/c5e6d14ea42cfa80/node_modules/@babel/parser/src/parser/statement.ts:359:17)
    at JSXParserMixin.parseBlockOrModuleBlockBody (/Users/jinma/.npm/_npx/c5e6d14ea42cfa80/node_modules/@babel/parser/src/parser/statement.ts:1241:25)
    at JSXParserMixin.parseBlockBody (/Users/jinma/.npm/_npx/c5e6d14ea42cfa80/node_modules/@babel/parser/src/parser/statement.ts:1216:10)
    at JSXParserMixin.parseBlockBody (/Users/jinma/.npm/_npx/c5e6d14ea42cfa80/node_modules/@babel/parser/src/plugins/estree.ts:161:13)
```
<img width="1150" alt="Screen Shot 2022-11-15 at 19 33 15" src="https://user-images.githubusercontent.com/9401060/201897937-83b6a91b-11b2-47b0-b666-072f65fce622.png">